modular/gater/bucket_handler.go

```diff
@@ -62,11 +66,27 @@ func (g *GateModular) getBucketReadQuotaHandler(w http.ResponseWriter, r *http.R
 		err = ErrConsensus
 		return
 	}
-	charge, free, consume, err = g.baseApp.GfSpClient().GetBucketReadQuota(
-		reqCtx.Context(), bucketInfo, reqCtx.vars["year_month"])
-	if err != nil {
-		log.CtxErrorw(reqCtx.Context(), "failed to get bucket read quota", "error", err)
-		return
+
+	log.CtxDebugw(reqCtx.Context(), "get bucket info when query quota:", "quota", bucketInfo.ChargedReadQuota)
+	bucketTraffic, err = g.baseApp.GfSpDB().GetBucketTraffic(
+		bucketInfo.Id.Uint64())
+	// if the traffic table has not been created and initialized yet, return the chain info
+	if errors.Is(err, gorm.ErrRecordNotFound) || bucketTraffic == nil {
+		free, err = g.baseApp.Consensus().QuerySPFreeQuota(reqCtx.Context(), g.baseApp.OperatorAddress())
+		if err != nil {
+			err = ErrConsensus
+		}
+		charge = bucketInfo.GetChargedReadQuota()
+		log.CtxDebugw(reqCtx.Context(), "fail to get traffic info in db, return the chain meta", "free quota", free, "charged quota", charge)
```

Comment:
`"free quota", free, "charged quota", charg` to `"free_quota", free, "charged_quota", charge`

Reply:
updated